### PR TITLE
chore: migrate clusters map for paris campus to noc42

### LINKED ISSUE #461: feat: cluster map updates for Paris campus (for new buildings)
### Is your feature request related to a problem? Please describe

The new buildings (BESS, PAUL) are open, thus the cluster map needs to be updated

### Describe the solution you'd like

The new map for Paris campus is up, and simply would like to update!

### Describe alternatives you've considered

_No response_

### Additional context

The `BESS` has one cluster name for each cluster that mainly takes up one whole floor, but `PAUL` has some sub-clusters that are divided into two clusters on the same floor
I had a conversation with @42atomys on Discord, and decided to go with the same as the below for now. (Start of the conversation : https://discord.com/channels/248936708379246593/954510970586284082/1117509039434244157 )
Below images are the cluster map image:

image
image
image
image
image
image
image

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by sungyongcho**
**Describe the pull request**

This pull request involves the migration of the clusters map for the Paris campus to noc42. The reason for this migration is due to construction work at 42 Paris, where the old building no longer exists, and has been replaced by noc42. This task ensures that the clusters map accurately represents the current state of the campus, providing an up-to-date resource for users. By completing this migration, we maintain the accuracy and relevance of our campus information, ensuring a smooth transition to the new noc42 setting.

**Checklist**

- [x] I have made the modifications or added tests related to my PR
- [x] I have run the tests and linters locally and they pass
- [x] I have added/updated the documentation for my RP

**Additional context**

Hello, paris cluster map upating pull request is here, but there are couple of things needs to be clarified, and needs modification

1. for creating a new Cluster object, it gives error when `ideitifier` is set to 'bess-f1' or 'paul-f1', even after updating extractorRegexp to below
```
  extractorRegexp = (): RegExp =>
    /(?<clusterWithLetter>(e|bess-f)(?<cluster>\d+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>p(?<workspace>\d+))/i;
```
the error arise from clusterMap.tsx file, and needs an investigations.

2. For bess-f1, the 'P' blocks are for personal computer space.

3. For Paul clusters, i went to the clusters, and it was physically separated room, and don't have a connection. My opinion is, since the two clusters are separated, it would be better to display them separately, but that's just my opinion.

I've made the codes for bess-f1,2,3 and paul-f3, and turned them into the comments, because of the `1.` giving out an error.

close #461

### CODE REVIEW

**Comment by 42atomys** [comment]
Hey,thanks for your inputs and context, i will check this out tomorrow with my morning coffee ☕ 

Thanks again for your contribution for Paris campus !

**Comment by sungyongcho** [comment]
Hello, for now all the cluster maps are added!
I've applied 'PW', the personal worplace, to bess-F1, for now. 
and I have a suggestion, that some of the cluster has meeting rooms, which are placed inside the cluster area (yes the new building feels new) that students can freely use. I wonder if I can use 'PW' for those areas, or maybe another notion(icon) is needed for it..?

(If this is not appropriate for PR, please let me know!!)

**Comment by 42atomys** [comment]
@sungyongcho I don't have this information as well, we can release this map without meeting room and see in another PR to add a solution for meeting room, WDYT ?

File: web/ui/src/grpc/jwtks/jwtks.ts
Comment (42atomys) [inline web/ui/src/grpc/jwtks/jwtks.ts:1]: This file is not present anymore, you can delete it and remove it from the pull request. 

I think this is a file from previous contribution :)

**Comment by sungyongcho** [comment]
hello @42atomys, I totally agree with your idea, and will bring you some photos next week! I'm not sure how to get inputs from the active user of the service tho...

**Comment by 42atomys** [comment]
> I'm not sure how to get inputs from the active user of the service tho...
I don't have tool to do that, maybe I send a survey on discord if some active users answer to the poll 

Maybe think about an "experiment" architecture to toggle (like feature flag) some part of the app in the future can be awesome ans useful. 

Note: I run the formatter for you, I will merge it 🎉 

Release v0.26 will be done tomorrow with morning coffee, including this changes ☕

**Pull request merged**